### PR TITLE
New package added

### PR DESCRIPTION
**Pull request by CRMTE1**
review

### CODE REVIEW

File: Project Dumy/src/test/java/TestNGPgm111/Batch22.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	jlib.verifyInfo(headerinfo,L_Name);
+	
+	// verify  LastName info 
+	String act_L_Name = lfip.getLeadslastNameInfo().getText();
+	jlib.verifyInfo(act_L_Name, L_Name);
+	
+	// verify CompanyName info 
+	String act_C_Name = lfip.getCompanyInfo().getText();
+	jlib.verifyInfo(act_C_Name, C_Name);
+
```
Comment (tekpramidSelenium) [inline Project Dumy/src/test/java/TestNGPgm111/Batch22.java:42]: don't create duplicate packages

**Review by tekpramidSelenium — COMMENTED** [review]
ok added

File: Project Dumy/src/test/java/TestNGPgm111/Batch22.java (hunk 1 above)
Comment (CRMTE1) [reply Project Dumy/src/test/java/TestNGPgm111/Batch22.java:42]: fixed

**Comment by tekpramidSelenium** [comment]
merged

**Pull request merged**